Gemfile

```diff
@@ -9,4 +9,4 @@ gem 'codecov', require: false
 gem 'webmock', require: false
 gem 'yard'
 
-gem 'rspec-buildkite-analytics', '~> 0.8.1'
+gem 'buildkite-test_collector', '~> 2.1'
```

Comment:
Would it be worth changing this to `2.2` since the gem in the lock file is ending up with `2.2`?

Reply:
Good point, updated in https://github.com/wordpress-mobile/release-toolkit/pull/465/commits/a06257eed46daa53078e90377021ca7240b5dfcf